src/sql/workbench/parts/notebook/browser/notebook.contribution.ts

```diff
@@ -292,4 +292,17 @@ registerComponentType({
 	selector: MarkdownOutputComponent.SELECTOR
 });
 
+/**
+ * A mime renderer for IPyWidgets
+ */
+registerComponentType({
+	mimeTypes: [
+		'application/vnd.jupyter.widget-view',
+		'application/vnd.jupyter.widget-view+json'
+	],
+	rank: 47,
```

Comment:
Why is this rank different from its defaultRank?

Reply:
I spot checked a few others and they were like that as well. I think there was already a 45/46 so I chose 47